src/wallet/rpcwallet.cpp

```diff
@@ -1764,20 +1764,14 @@ UniValue walletconfirmbackup(const UniValue& params, bool fHelp)
     EnsureWalletIsUnlocked();
 
     SecureString strMnemonicPhrase(params[0].get_str());
-    boost::erase_all(strMnemonicPhrase, "\"");
```

Comment:
Now that usage of this function has been removed, remove the corresponding header import and lint allowlist entry.